src/main/java/duke/Save.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+
+class Save {
+    /**
+     * Initliaze ArrayList of Task.
+     * if file "save" does not exist, create it and return empty ArrayList of Task
+     * else load contentes of "save" into the ArrayList of Task
+     * @return ArrayList of Task
+     */
+    public static ArrayList<Task> loadSave() {
+        File file = new File("./save");
+        if (file.exists()) {
+            try (FileInputStream fis = new FileInputStream("./save");
+                    ObjectInputStream ois = new ObjectInputStream(fis);
```

Comment:
Maybe something to note, ObjectInputStream has some security vulnerabilities: https://owasp.org/www-community/vulnerabilities/Deserialization_of_untrusted_data. No issue here, just for further reading!